### PR TITLE
Add sniff for empty format string

### PR DESCRIPTION
**Pull request by lstrojny**

### CODE REVIEW

File: src/InterNations/Sniffs/Waste/SuperfluousFormatStringSniff.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        $openingBrace = $tokens[$stackPtr + 1];
+
+        for ($pos = $openingBrace['parenthesis_opener'] + 1; $pos < $openingBrace['parenthesis_closer']; $pos++) {
+            if (!in_array($tokens[$pos]['code'], [T_CONSTANT_ENCAPSED_STRING, T_WHITESPACE, T_STRING_CONCAT], true)) {
+                return;
+            }
+        }
+
+        $file->addError(
+            'Superfluous %s() call as no parameters are passed',
```
Comment (ajb-in) [inline src/InterNations/Sniffs/Waste/SuperfluousFormatStringSniff.php]: Can the error message suggest what should be done instead?

Comment (lstrojny) [inline src/InterNations/Sniffs/Waste/SuperfluousFormatStringSniff.php]: Done

File: src/InterNations/Sniffs/Waste/SuperfluousFormatStringSniff.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        $openingBrace = $tokens[$stackPtr + 1];
+
+        for ($pos = $openingBrace['parenthesis_opener'] + 1; $pos < $openingBrace['parenthesis_closer']; $pos++) {
+            if (!in_array($tokens[$pos]['code'], [T_CONSTANT_ENCAPSED_STRING, T_WHITESPACE, T_STRING_CONCAT], true)) {
+                return;
+            }
+        }
+
+        $file->addError(
+            'Superfluous %s() call as no parameters are passed. You can safely remove the function call',
```
Comment (ajb-in) [inline src/InterNations/Sniffs/Waste/SuperfluousFormatStringSniff.php]: Maybe that's true for `sprintf`, but what about `printf`?

Comment (lstrojny) [inline src/InterNations/Sniffs/Waste/SuperfluousFormatStringSniff.php]: Good point. Fixed in master.

**Pull request merged**